week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,31 +1,49 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
+//Math.random returns a random number bewteen 0 and 10.
+//Therefore, this function will return the result of a random number multiplied by 10.
+
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+
+//This function will join the two strings (w1 and w2) together.
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
+// Write the body of this function to concatenate three words together
+// Look at the test case below to understand what to expect in return
+
 function concatenate(firstWord, secondWord, thirdWord) {
-    // Write the body of this function to concatenate three words together
-    // Look at the test case below to understand what to expect in return 
+  return firstWord + " " + secondWord + " " + thirdWord;
+  //   return `${firstWord} ${secondWord} ${thirdWord}`;
```

Comment:
Here it is not correct what you did, you have been asked to use concate method here. Fix that please.